docs/concepts.rst

```diff
@@ -271,3 +271,89 @@ Modules not yet Supported for Criteria
 :mod:`RunHistory <cbc_sdk.audit_remediation.base.RunHistory>`
 :mod:`Event <cbc_sdk.enterprise_edr.base.Event>`
 :mod:`Process <cbc_sdk.enterprise_edr.base.Process>`
+
+
+Asynchronous queries
+--------------------
+
+A number of queries allow for asynchronous mode of operation. Those utilize python futures and the request itself is performed in a separate worker thread.
+An internal thread pool is utilized to support multiple CBC queries executing in an asynchronous manner without blocking the main thread.
+
+Execute an asynchronous query
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Running asynchronous queries is done by invoking the ``execute_async()`` method, e.g:
+
+  >>> async_query = api.select(EnrichedEvent).where('process_name:chrome.exe')
```

Comment:
You didn't include the execute_async() in the example

Reply:
Oops...fixed